components/cluster/command/check.go

```diff
@@ -442,6 +442,11 @@ func fixFailedChecks(ctx *task.Context, host string, res *operator.CheckResult,
 			),
 			true)
 		msg = fmt.Sprintf("will try to %s, reboot might be needed", color.HiBlueString("disable SELinux"))
+	case operator.CheckNameTHP:
+		t.Shell(host,
+			"echo never > /sys/kernel/mm/transparent_hugepage/enabled && echo never > /sys/kernel/mm/transparent_hugepage/defrag",
```

Comment:
This will disable THP in runtime, if the system rebooted, the THP's status maybe enabled again? Should we disable it permanently? ref https://access.redhat.com/solutions/46111

Reply:
Since the origin issue is reference to tidb-ansible, maybe just keep the same behavior as [tidb-ansible](https://github.com/pingcap/tidb-ansible/pull/1079) is enough

Reply:
It seems this config does not have an entry in `sysctl`, so the only way to make it consist after reboot would be adding a kernel parameter at boot. ref https://www.kernel.org/doc/html/v5.4/admin-guide/mm/transhuge.html#boot-parameter

Reply:
It's  unpractical to add a kernel parameter, just remind the user to run `tiup cluster check` again after reboot.